vuepress/docs/next/tutorials/curate/bundle-private-images.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebarDepth: 2
+---
+
+# Install Bundle from a Private Image Repository
+
+This tutorial provides to way to utilize bundles from a private image repository in your Entando projects. Private repositories require user authentication and the steps below use environment variables to pass the Secret for authentication.
+
+**New or Existing Bundle**
+* If you have an existing bundle in a private image repository, start at Step 1: Create the Registry Credentials. 
+* If you are starting a new project, start at [Step 2: Create and Apply the Secret](#step-2-create-and-apply-the-secret).
+
+## Prerequisites
+
+* [A working instance of Entando](../../../docs/getting-started/)
+* Verify dependencies with the [Entando CLI](../../docs/getting-started/entando-cli.md#check-the-environment): `ent check-env develop`
+
+### Step 1: Create the Registry Credentials
+1. Create the registry JSON configuration using your registry and credentials on the port of your choice: 
+``` json
+{
+  "auths": {
+    "YOUR-REGISTRY.com": {
+        "username": "YOUR-USERNAME",
+        "password": "YOUR-PASSWORD"
+    }
+  }
+}
+```
+
+2. Convert the JSON configuration into a base64 string  
+
+### Step 2: Create and Apply the Secret
+1. Create a `container-registry-secret.yaml` in your namespace with the following snippet. Make sure to replace the registryCredential value with your own. 
```

Comment:
you create a file, or you create a specific thing (not one instance of the same thing), so either "create a `container...yaml` file" or "create a file named `container...yaml`" or "create `container...yaml`"

Reply:
understood. it's tricky. whether or not the article applies depends on scope and context. strictly speaking there are definitely rules surrounding it, but those can be broken for clarity